signup/templates/signup/index.html

```diff
@@ -6,7 +6,13 @@
 <div class="container">
     <div class="row justify-content-center">
         <div class="col-6">
-            <h2>Sign Up for an Account</h2>
+            <h2>Sign Up for an Account<button onclick="goBack()">Back</button></h2>
```

Comment:
Same comment here as above, please move the button down opposite the signup button and make sure to match styles.